iothub/service/src/Utilities/Utils.cs

```diff
@@ -1,14 +1,13 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
 using System.Reflection;
 
 namespace Microsoft.Azure.Devices
 {
     internal class Utils
     {
-        public static string GetClientVersion()
+        internal static string GetClientVersion()
```

Comment:
We've decided to make public methods on internal classes internal instead. They aren't really public and it can be confusing in code reviews.